### PR TITLE
feat: [PE-413] Add icon button to copy EYCA card number and change of EYCA refs website

### PR DESCRIPTION
**Pull request by Hantex9**
## Short description
This PR upgrades the EYCA visibility discounts, adding a CTA button to the EYCA discounts in the CGN details page, adding a new icon button to copy card number, changed some text for the EYCA details bottom sheet to be more clear to the user with inside a CTA to the EYCA website.

## Design references
[FIGMA](https://www.figma.com/file/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale_20220211?type=design&node-id=2495-66351&t=Lej8FSywOGzEzaQd-0)

## List of changes proposed in this pull request
- Edited some existing translations and added a new `showEycaDiscounts` translation key
- Changed the behavior of the "Copia numero carta EYCA" button into a CTA "Vedi le agevolazioni EYCA" that redirects to the EYCA discounts;
- Added a `IconButton` to copy the EYCA card number;
- Changed text description into EYCA bottom sheet;
- Changed CTA link into EYCA description bottom sheet button that redirects to the EYCA website instead of EYCA discounts;
- Migrated `EYCA_DISCOUNTS_URL` and `EYCA_WEBSITE_URL` constants into a constants file of CGN feature directory

## How to test
- Login into app;
- Add a new CGN card;
- Open Wallet section;
- Tap on "Carta Giovani Nazionale" card and go into CGN details page 

## Preview

https://github.com/pagopa/io-app/assets/34343582/b34bcbde-dc74-40c3-8292-7363990b40e0

### CODE REVIEW

File: ts/features/bonus/cgn/components/detail/eyca/EycaInformationComponent.tsx [hunk 1]
```diff
@@ -8,8 +8,7 @@ import ButtonDefaultOpacity from "../../../../../../components/ButtonDefaultOpac
 import { Label } from "../../../../../../components/core/typography/Label";
 import { showToast } from "../../../../../../utils/showToast";
 import { VSpacer } from "../../../../../../components/core/spacer/Spacer";
-
-const EYCA_DISCOUNTS_URL = "https://www.eyca.org/discounts/it";
+import { EYCA_WEBSITE_URL } from "../../../utils/constants";
```
Comment (francescopersico) [inline ts/features/bonus/cgn/components/detail/eyca/EycaInformationComponent.tsx:11]:
```suggestion
import { EYCA_WEBSITE_DISCOUNTS_PAGE_URL } from "../../../utils/constants";
```
What you think about this? Isn't it a specific page/section of the website?

Comment (francescopersico) [reply ts/features/bonus/cgn/components/detail/eyca/EycaInformationComponent.tsx:11]: You can also have the base website url and the discount page one relative to the base one.

File: ts/features/bonus/cgn/components/detail/eyca/EycaStatusDetailsComponent.tsx [hunk 2]
```diff
@@ -109,10 +126,14 @@ const EycaStatusDetailsComponent = (props: Props) => {
         bordered
         style={{ width: "100%" }}
         onPress={() =>
-          clipboardSetStringWithFeedback(props.eycaCard.card_number)
+          openWebUrl(EYCA_DISCOUNTS_URL, () =>
```
Comment (francescopersico) [inline ts/features/bonus/cgn/components/detail/eyca/EycaStatusDetailsComponent.tsx:129]: Is this change, from copy to clipboard to openurl, by design?

File: ts/features/bonus/cgn/utils/constants.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const EYCA_DISCOUNTS_URL = "https://eyca.org/discounts/it";
```
Comment (francescopersico) [inline ts/features/bonus/cgn/utils/constants.ts:1]: This can be composed from the EYCA_WEBSITE_URL.

**Review by francescopersico — CHANGES_REQUESTED** [review]

File: ts/features/bonus/cgn/components/detail/eyca/EycaInformationComponent.tsx (hunk 1 above)
Comment (Hantex9) [reply ts/features/bonus/cgn/components/detail/eyca/EycaInformationComponent.tsx]: Previously in that page the link was to EYCA discounts page but by design is changed redirecting to the EYCA **homepage** website. 

I agree with you to build the discount page from website url as base path and I've done that with this commit: [122d472](https://github.com/pagopa/io-app/pull/4689/commits/122d472cd96d6010c134351178fba3259c60a7d2)

File: ts/features/bonus/cgn/utils/constants.ts (hunk 3 above)
Comment (Hantex9) [reply ts/features/bonus/cgn/utils/constants.ts]: That's absolutely better, thank you. 🙏

Done as you suggested with this commit: [122d472](https://github.com/pagopa/io-app/pull/4689/commits/122d472cd96d6010c134351178fba3259c60a7d2)

**Review by francescopersico — APPROVED** [review]

**Pull request merged**